doc/usage.rst

```diff
@@ -60,80 +60,81 @@ On the bottom left is a `google` icon.
 .. image:: google.png
 
 If you click on the `google` icon, you will be able to see a bigger version
-of the image, allowing to see it in more detail.
+of the image, allowing you to see it in more detail.
 
 On the bottom right is a `check` icon.
 
 .. image:: checked.png
 
-If you click on this `checked` icon, you will select this image to vote on.
+If you click on this `check` icon, the corresponding image will be selected to be voted upon.
 
 As you select candidates, these candidates will disapear from the central
-list of candidates and move to the grey column on the right where are
-presented all the candidates you selected.
+list of candidates and move to the grey column on the right where all the candidates you selected
+are shown.
 
-At the top of this selection, still in the grey column on the right is
+At the top of this selection, within the grey column on the right, is
 displayed the number of candidates you selected versus the maximum number of
 candidates you are allowed to select.
 
-Once you selection is made, simply click on ``Submit Votes`` on the grey
+Once your selection is made, simply click on ``Submit Votes`` on the grey
 column on the right.
 
-You will be brought back to the list of elections and thanks for voting.
+You will be brought back to the list of elections and with a message thanking you for voting.
 
 .. note:: You may vote on as many or as little candidates as you like up
           to the maximum allowed per elections.
           If you select less than the maximum number of candidates you will
           be able to come back on the election page and complete your vote
-          (thus allowing to vote in several times).
+          (thus allowing you to vote several times).
 
 .. warning:: You cannot change your vote once you have submitted it.
 
 
 
-Finding published elections
+Finding Published Elections
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Published elections are elections whose resules have been made public.
+Published elections are elections whose results have been made public.
 
 To find all the elections whose results are public, go to the ``Elections``
 tab at the top of the page.
 
-In the column ``Published`` of the table the election whose results are
-public are marked with:
+The election whose results arepublic are marked with:
 
 .. image:: Approved.png
 
-While elections whose results are not yet public are marked with the symbol:
+Under the column ``Published`` in the table on this page.
+
+The elections whose results are not yet public are marked with the symbol:
```

Comment:
I kind of liked specifying the column earlier as it is true for both symbols, which it lost in your wording.